src/DIRAC/ConfigurationSystem/scripts/dirac_admin_add_resources.py

```diff
@@ -30,21 +30,25 @@
 
 def processScriptSwitches():
 
-    global vo, dry, doCEs, hostURL, glue2
+    global vo, dry, doCEs, hostURL, glue2, onecore
 
     Script.registerSwitch("V:", "vo=", "Virtual Organization")
     Script.registerSwitch("D", "dry", "Dry run")
     Script.registerSwitch("C", "ce", "Process Computing Elements")
     Script.registerSwitch("H:", "host=", "use this url for information querying")
     Script.registerSwitch("G", "glue2", "DEPRECATED: query GLUE2 information schema")
     Script.registerSwitch("g", "glue1", "query GLUE1 information schema")
+    Script.registerSwitch(
+        "1", "onecore", "Add Single Core Queues for each MultiCore Queue, set RequiredTag for those Queues"
```

Comment:
```suggestion
        "", "onecore", "Add Single Core Queues for each MultiCore Queue, set RequiredTag for those Queues"
```

Reply:
What is wrong with the `-1` flag?

Reply:
I don't like using numerals for a purpose that is not numeric. Just a personal feeling, but I find it confusing.

Reply:
OK, dropped the `-1` , it does look weird